### DISCUSSION TITLE
Connect third party application on network level

### DISCUSSION THREAD

**Discussion by MerouaM** [magma/magma#14890] [Q&A]
Hello everyone,

I would like to connect a third party application to Magma core, however, I would like to do that on "network" level rather than on "application" level. Is this possible ?

Thank you,

Best.

**Comment by jordanvrtanoski** [comment]
Can you give us more details of the scenario you have in mind? What would you like the application to do with Magma? What would be the role of Magma?

**Comment by panyogesh** [comment]
Closing the discussion as it seems quite old. Kindly raise a new issue or discussion topic if issue is still observed.
